README.md

```diff
@@ -6,6 +6,26 @@ The standard object structure is given at https://docs.google.com/document/d/1Jm
 
 The yaml style can be reproduced with `js-yaml` (https://www.npmjs.com/package/js-yaml) using the dump options `{ sortKeys: true, lineWidth: Infinity }`.
 
+## Special Markdown Syntax
+
+Anywhere `markdown` is allowed in a yaml, the following syntax is allowed:
+```md
+[displayed link text](source:file/key)
+[[displayed popup text]](source:file/key)
```

Comment:
For popup text, I'd like to request that we use pretty much anything other than `[[` and `]]`. It doesn't work with normal Markdown parsers, and I don't want to have to write an entirely new one just for this feature. Something like `[{link text}]`, however, would be fine.

Reply:
Skill issue

but ok

Reply:
Jokes aside, I would like whatever link format we use to ultimately still be recognized as a valid link in Markdown. Double brackets are not, unfortunately.